.changeset/mighty-crews-vanish.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+fix: support minimum gas price in args
```

Comment:
fix: allow setting min gas price in `launchNode` args

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2814/commits/81eafa3c89827cfa498b0a93404d350ed1bbdf00